src/main/java/com/devcycle/sdk/server/cloud/api/DVCCloudClient.java

```diff
@@ -46,13 +46,28 @@ public Map<String, Feature> allFeatures(User user) throws DVCException {
     return getResponse(response);
   }
 
+  /**
+   * Get variable value by key for user data
+   *
+   * @param user (required)
+   * @param key  Feature key (required)
+   * @param defaultValue Default value to use if the variable could not be fetched
+   *                     (required)
+   * @return Variable value
+   * @throws DVCException If there are any uses with the data provided
+   */
+  public <T> T variableValue(User user, String key, T defaultValue)  throws DVCException {
+    return variable(user, key, defaultValue).getValue();
+  }
+
   /**
    * Get variable by key for user data
    * 
    * @param user  (required)
    * @param key Variable key (required)
    * @param defaultValue Default value to use if the variable could not be fetched (required)
    * @return Variable
+   * @throws DVCException If there are any uses with the data provided
```

Comment:
Not for this PR, but we should talk about if our SDKs should throw at all or just always return default.

Reply:
Agreed, I was really surprised that the cloud SDK throws exceptions -- though it only does it if one of the parameters provided is invalid